### PR TITLE
[CI]Building and testing Java with Maven

### PR DESCRIPTION
**Pull request by CalvinKirs**

### CODE REVIEW

File: .github/workflows/backend.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Java CI
+
+on:
+  push:
```
Comment (zhongjiajie) [inline .github/workflows/backend.yml:21]:
```suggestion
  push:
  pull_request:
```

Comment (zhongjiajie) [reply .github/workflows/backend.yml:21]: I think we should requests all our PR to branch `wd_v2_baseline` and commit to any branch run CI, WDYT?

**Review by zhongjiajie — CHANGES_REQUESTED** [review]

**Comment by zhongjiajie** [comment]
For now, we just requests commit to branch `wd_v2_baseline` run CI, but not including PR to this branch.

**Comment by zhongjiajie** [comment]
@CalvinKirs Could you rebase? seem too many unrelated change here

**Comment by zhongjiajie** [comment]
It there any way to add rebase bottom to GitHub, which allow maintainer rebase on target instead of ask contributor rebase locally?

File: .github/workflows/backend.yml (hunk 1 above)
Comment (CalvinKirs) [reply .github/workflows/backend.yml:21]: yup,thx fixd

**Comment by CalvinKirs** [comment]
> It there any way to add rebase bottom to GitHub, which allow maintainer rebase on target instead of ask contributor rebase locally?

Well, let me take a look:)

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Comment by zhongjiajie** [comment]
Thanks Calvin for our CI, and if you could find a way to add button to to rebase to target branch it will better.

**Pull request merged**